index.css

```diff
@@ -773,26 +736,25 @@ body {
 form {
   display: flex;
   flex-direction: column;
+  width: 100%;
 }
 
 footer {
   border-top: 1px solid #dfe1e6;
   display: flex;
-  position: absolute;
-  width: 375px;
-  height: 72px;
+  width: 100%;
   left: 0;
 
   /* no unit */
-  top: 6973px;
   margin: 124px 0 10px 0;
 
   /* no unit */
   justify-content: center;
 }
 
 .footer_social_media {
-  align-self: center;
+  display: flex;
+  justify-content: center;
 }
```

Comment:
- [optional] Kindly increase the size of the social media handles, I hardly see it. 👍 

## Your Design
 
![socialmedia](https://user-images.githubusercontent.com/103386183/206431886-9739c263-7c40-4303-bf51-72fabf7033d3.png)